model/src/test/kotlin/utils/LicenseResolverTest.kt

```diff
@@ -246,10 +242,8 @@ class LicenseResolverTest : WordSpec() {
 
                 result should containExactlyInAnyOrder("Apache-2.0", "MIT")
             }
-        }
 
-        "Given a package with multiple scan results for VCS and source artifact, getDetectedLicensesForId()" should {
-            "return all detected licenses" {
+            "return all detected licenses, for a package with multiple scan results for VCS and source artifact" {
```

Comment:
The comma before "for" should be dropped here like in most other test names. And the second "for" should probably be reworded.

Reply:
Done.Done.